frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/components/AvailabilityOptionsForm.tsx

```diff
@@ -0,0 +1,138 @@
+    return (
+      <>
+        {t('Highly available')}{' '}
+        <PopoverIcon
+          bodyContent={t(
+            'Highly available means components should be resilient to problems across fault boundaries as defined by the component to which the policy is attached. This usually means running critical workloads with 3 replicas and with little or no toleration of disruption of the component.'
+          )}
+        />
+      </>
+    )
+  }
+
+  const labelSingle = () => {
+    return (
+      <>
+        {t('Single replica')}{' '}
+        <PopoverIcon
+          bodyContent={t(
+            'Single replica means components are not expected to be resilient to problems across most fault boundaries associated with high availability. This usually means running critical workloads with just 1 replica and with toleration of full disruption of the component.'
+          )}
+        />
+      </>
+    )
+  }
+
+  return (
+    <AcmForm className={formWrapper}>
+      <FormGroup role="radiogroup" isInline fieldId="controller" label={t('Controller availability policy')} isRequired>
+        <Radio
+          id={'controller-ha'}
+          data-testid="controller-ha"
+          name={'controllerAvailabilityPolicy'}
+          label={labelHA()}
+          value={'HighlyAvailable'}
+          defaultChecked={control.active.controller === 'HighlyAvailable'}
+          onChange={handleChange}
+        />
+        <Radio
+          id={'controller-single'}
+          data-testid="controller-single"
+          name={'controllerAvailabilityPolicy'}
+          label={labelSingle()}
+          value={'SingleReplica'}
+          defaultChecked={control.active.controller === 'SingleReplica'}
+          onChange={handleChange}
+        />
+      </FormGroup>
+      <FormGroup
+        role="radiogroup"
+        isInline
+        fieldId="infrastructure"
+        label={t('Infrastructure availability policy')}
+        isRequired
+      >
+        <Radio
+          id={'infra-ha'}
+          data-testid="infra-ha"
+          name={'infraAvailabilityPolicy'}
+          label={labelHA()}
+          value={'HighlyAvailable'}
+          defaultChecked={control.active.infra === 'HighlyAvailable'}
+          onChange={handleChange}
+        />
+        <Radio
+          data-testid="infra-single"
+          id={'infra-single'}
+          name={'infraAvailabilityPolicy'}
+          label={labelSingle()}
+          value={'SingleReplica'}
+          defaultChecked={control.active.infra === 'SingleReplica'}
+          onChange={handleChange}
+        />
+      </FormGroup>
+    </AcmForm>
+  )
+}
+
+export default AvailabilityOptionsForm
+
+export const summary = (control: any, t: TFunction) => {
+  const { controller, infra } = control.active || {}
+  const controlDesc: string = controller === 'HighlyAvailable' ? t('Highly Available') : t('Single Replica')
+  const infraDesc: string = infra === 'HighlyAvailable' ? t('Highly Available') : t('Single Replica')
```

Comment:
Exact same code repeated twice (and casing issue). Please extract to helper function.